invenio_app_rdm/translations/messages.pot

```diff
@@ -1,57 +1,108 @@
 # Translations template for invenio-app-rdm.
-# Copyright (C) 2019 CERN
+# Copyright (C) 2020 CERN
 # This file is distributed under the same license as the invenio-app-rdm
 # project.
-# FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2020.
 #
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: invenio-app-rdm 1.0.0a5\n"
+"Project-Id-Version: invenio-app-rdm 0.10.1\n"
 "Report-Msgid-Bugs-To: info@inveniosoftware.org\n"
-"POT-Creation-Date: 2019-12-19 08:33-0600\n"
+"POT-Creation-Date: 2020-07-07 13:39+0300\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Generated-By: Babel 2.7.0\n"
+"Generated-By: Babel 2.8.0\n"
 
-#: invenio_app_rdm/config.py:120
+#: invenio_app_rdm/config.py:128
 msgid "The turn-key research data management repository"
 msgstr ""
 
-#: invenio_app_rdm/config.py:129
-msgid "InvenioRDM: Turn-key Research Data Management Repository"
+#: invenio_app_rdm/config.py:137
+msgid "InvenioRDM"
 msgstr ""
 
-#: invenio_app_rdm/config.py:169
+#: invenio_app_rdm/config.py:176
 msgid "Welcome to Invenio App RDM!"
 msgstr ""
 
-#: invenio_app_rdm/theme/templates/invenio_app_rdm/footer.html:19
+#: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/footer.html:12
 #, python-format
-msgid "Powered by <a href=\"%(invenio_rdm)s\">InvenioRDM</a>"
+msgid ""
+"Powered by <a\n"
+"        href=\"%(invenio_rdm)s\">InvenioRDM</a>"
+msgstr ""
+
+#: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/frontpage.html:9
+msgid "You've successfully installed InvenioRDM! What is ahead?"
+msgstr ""
+
+#: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/frontpage.html:17
+msgid "Configure"
+msgstr ""
+
+#: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/frontpage.html:19
+#, python-format
+msgid ""
+"Configure the\n"
+"            datamodel, resource types, permissions and much more to make "
+"it suit your needs! For more\n"
+"            information on how to do it visit the <a "
+"href=\"%(configuration_link)s\">configuration\n"
+"              section in the docs</a>."
+msgstr ""
+
+#: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/frontpage.html:29
+msgid "Customize"
+msgstr ""
+
+#: invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/frontpage.html:32
+#, python-format
+msgid ""
+"Customize\n"
```

Comment:
@lnielsen do you think of a way to avoid all these newline symbols injected in the collected string?

Reply:
+1 yeah it's weird and seems fragile.

When working with transifex and translations in the past we had a similar situation without clear solution...

Another take is the Android approach: every string to be translated is just a placeholder/symbol. It makes it harder to understand the text in context (i.e. in the frontpage you would see `_("frontpage_step2_customize")` instead of this full text), but then changes in text are less brittle. I also suspect there might be vscode extensions that can translate/expand back in your editor.